packages/python/accern_data/util.py

```diff
@@ -43,6 +45,14 @@ def is_test() -> int:
     return IS_TEST
 
 
+def set_data_dir(path: str) -> None:
```

Comment:
why not
```
def set_data_dir(path: str) -> None:
    global DATA_DIR
    
    DATA_DIR = path
```
?

Reply:
global would access and overwrite variable defined in other files?

Reply:
no